code/atom.dm

```diff
@@ -206,8 +208,14 @@
 	*
   * * returns true if open, false if closed
 	*/
-	proc/is_open_container()
-		return flags & OPENCONTAINER
+	proc/is_can_receive()
+		return rc_flags & CAN_RECEIVE
+
+	proc/is_can_transfer()
+		return rc_flags & CAN_TRANSFER
+
+	proc/is_can_splash()
+		return rc_flags & CAN_SPLASH
```

Comment:
`is_can` sounds incredibly wordy and poor english

Reply:
will a simple can_() work?